.drone.yml

```diff
@@ -117,6 +115,11 @@ environment:
   DOCKER_HOST: tcp://docker:2375
 
 steps:
+  - name: fetch and checkout
+    image: alpine/git
+    commands:
+      - git fetch --tags
+      - git checkout $${VERSION}
```

Comment:
Will this break if we use `build_123`?

Reply:
Yes this would break - we are no longer using build_ and now using the commit SHAs. We don't push the build_ tag anymore.